tests/test_finders.py

```diff
@@ -85,31 +93,30 @@ def test_whisper_finder(self):
                 os.makedirs(os.path.dirname(db_path))
             whisper.create(db_path, [(1, 60)])
 
-        def listdir_mock(d):
-            self._listdir_counter += 1
-            return self._original_listdir(d)
+        def scandir_mock(d):
+            self._scandir_counter += 1
+            return self._original_scandir(d)
 
         try:
-            os.listdir = listdir_mock
+            scandir.scandir = scandir_mock
```

Comment:
This would be better handled with `unittest.mock` (or `mock` on older Python versions). You can have a look at `test_carbonlink.py` for usage examples. In particular [call_count](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_count) would be useful.